src/components/ItaliaTheme/View/Commons/PageHeader/PageHeaderEventDates.jsx

```diff
@@ -36,26 +36,31 @@ const PageHeaderEventDates = ({ content, moment, rrule }) => {
 
```

Comment:
Da un rapido check mi sembra che un pezzo di roba simile esista anche in https://github.com/RedTurtle/design-comuni-plone-theme/blob/11.x.x/src/components/ItaliaTheme/View/Commons/Dates.jsx.

Valutare se va allineato anche questo pezzetto di codice. Se va allineato, crea un helper a parte che evitiamo questo tipo di possibili sviste con una sola funzione riusabile.
Bonus: proprio per quel componente potrebbero esserci noie, sono appena riuscita a romperlo qui, magari dacci un occhio se alla fine vedi ci devi passare, altrimenti solita procedura.
ref: https://v3.io-comune.redturtle.it/test-martina/evento-2

Reply:
Nel componente dates non viene gestita la stringa "dal al", c'è direttamente il calendario che funzionava già correttamente
Gestito l'errore che vedevi tu, si generava con il caso  ricorrenza "termina dopo x occorrenze" quando c'era una data di inizio !== data fine